github/ci/services/grafana/manifests/grafana.yaml

```diff
@@ -8091,13 +8093,13 @@ data:
                 "uid": "99254836-eba2-4667-aa63-763bfacabaf1"
               },
               "editorMode": "code",
-              "expr": "rate(referee_retests_kubevirt_kubevirt_total[1h])",
-              "legendFormat": "retest rate",
+              "expr": "increase(referee_retests_kubevirt_kubevirt_total[1h])",
+              "legendFormat": "retests/h",
               "range": true,
               "refId": "A"
             }
           ],
-          "title": "kubevirt/kubevirt: retest rate / h",
+          "title": "k/kubevirt: retests/h",
```

Comment:
I'm not sure about the title change `k/kubevirt` doesn't exist. I would prefer kubevirt/kubevirt or simply just kubevirt. 
```suggestion
          "title": "kubevirt: retests/h",
```